explainers/console.context()/explainer.md

```diff
@@ -0,0 +1,49 @@
+# DevTools: console.context()
+
+Authors:
+ - *[Leah Tu](https://github.com/leahmsft)*, Microsoft Edge
+ - *[Patrick Brosset](https://github.com/captainbrosset)*, Microsoft Edge
+
+
+## Status of this feature
+
+This is available as of Microsoft Edge 79.
+
+## Introduction
+
+While debugging a large application, filtering through many console log messages can be challenging. If a developer wanted to group messages together, they could use console.group(), but it's possible that other unrelated logs could be included. Alternatively, they could append a group name before log messages, but this is tedious and time-consuming.
+
+## Goals
+The console.context() method provides a better solution for managing log messages. It returns a console instance that implements the console interface. Messages can be organized by logging to different contexts.
```

Comment:
```suggestion
Our goal is to improve on the existing experimental `console.context()` method to provide a better solution for logging messages from a multi-component web app's code base, and more easily filter messages in the Console.

The `console.context(contextName)` method returns an instance of an object that implements the same methods as the `console` namespace. Developers can create different contexts for the different parts of their apps. Messages logged from a context object _belong_ to the context and can be filtered in the Console tool, currently by typing `context:contextName` in the Console's search field.
```

Reply:
We will need a new section at the bottom of this explainer that describes the interface too. The context method is not standard right now (not in https://console.spec.whatwg.org/), so we need to write some IDL code to propose what the method would return, etc.